src/plugins/data/public/autocomplete/providers/value_suggestion_provider.ts

```diff
@@ -19,13 +19,14 @@
 
 import { memoize } from 'lodash';
 import { CoreSetup } from 'src/core/public';
-import { IIndexPattern, IFieldType, UI_SETTINGS } from '../../../common';
+import { IIndexPattern, IFieldType, UI_SETTINGS, buildQueryFromFilters } from '../../../common';
+import { getQueryService } from '../../services';
 
-function resolver(title: string, field: IFieldType, query: string, boolFilter: any) {
+function resolver(title: string, field: IFieldType, query: string, filters: any[]) {
```

Comment:
Can this be `filters: Filter[]` in this whole file?

Reply:
These are not filters of type `Filter`, but an array of whatever type `BoolQuery.filter` is ( `src/plugins/data/server/search/es_search/elasticsearch.ts`).

We could fix that, but I wouldn't want to do it in this PR.